src/config/reddit/config_gen.py

```diff
@@ -13,11 +13,23 @@
 def config_gen():
     if not os.path.isfile(ENV_FILE):
       # ASK FOR CREDENTIALS
-      
-      CLIENT_ID = input('please input your account client id :')
-      CLIENT_SECRET = input('please input your account client secret :')
-      PASSWORD = input('please input your account password :')
-      USERNAME = input('please input your account username :')
+    def CI():
+        CLIENT_ID = input('please input your account client id :')
+        if len(CLIENT_ID) == 14 :
+            pass
+        else:
```

Comment:
if there's an error i think it should log to the user that something bad happened. so `log.info("client id must be 14 characters, try again")`. also it seems the [praw documentation](https://praw.readthedocs.io/en/latest/getting_started/authentication.html#password-flow) reports different length values for them. so that's very curious